### ISSUE TITLE
translation into Russian, encoding

### ISSUE THREAD

**Issue by GreenL1nk** [lokka30/PhantomEconomy#20]
instead of the Russian language, these characters
![image](https://user-images.githubusercontent.com/47003416/78195904-86417600-7489-11ea-8194-26beafd17c4e.png)

**Comment by GreenL1nk** [comment]
several Russian characters for the test "абвгд" (if u need)

**Comment by dkbay** [comment]
@GreenL1nk Hey!

I had the same issue for æøå I ended up asking on the Paper Discord and they told me to add 
`-Dfile.encoding=UTF-8` to my start.sh/start.bat make sure to add it just before -jar

Just like this
![image](https://user-images.githubusercontent.com/1548975/78212780-00d1bc00-74b1-11ea-8044-1666be1d573a.png)

There might be a way of adding this to the plugin but until then... 
I think the issue could be with LightningStorage, as I could see the letters fine in the file, and if I wrote them into a variable directly they also show fine. It only seems to break when I try to load it through LightningStorage.

**Comment by GreenL1nk** [comment]
@dkbay thank you, it worked

**Comment by lokka30** [comment]
@dkbay Thank you very much for providing the fix - I've had quite a few people ask me why special characters aren't working as intended. I always knew that the issue was definitely not inside my code, however from a library, or in this case, startup arguments.
I've spoken with the developer of LightningStorage, and they've stated that special characters work perfectly fine for them.
Looks like the only way to fix this issue is by adding the startup argument.

**Comment by dkbay** [comment]
I spent some time reading into this, and it depends on what OS you're running since the JVM will get the charset from there unless you set that startup arg...
So on some OS' like Ubuntu 18.04 which I run for my server in production, it'll run just fine without any startup arg cause the default charset is `C.UTF-8`, but on other OS like Windows 10 it'll use `Windows-1252` 

TLDR; Some people will run into it others won't. It depends on the OS settings, the startup arg fixes it if it doesn't work by default.

I hope this clears it up a bit more :D 

Sources:
https://www.baeldung.com/java-char-encoding
https://unix.stackexchange.com/questions/251331/what-is-the-default-encoding-for-javac-in-linux

**Comment by lokka30** [comment]
Thanks!
